### PR TITLE
megatokenizer-regression-docs

### PR DESCRIPTION
**Pull request by theyorubayesian**
Add regression docs for megatokenizer & composite tokenizer

### CODE REVIEW

File: src/main/resources/docgen/templates/msmarco-doc-composite-tokenizer.template [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Anserini Regressions: MS MARCO Document Ranking
```
Comment (lintool) [inline src/main/resources/docgen/templates/msmarco-doc-composite-tokenizer.template:1]: There is no corresponding yaml here? Don't understand what this template is for?

Comment (lintool) [reply src/main/resources/docgen/templates/msmarco-doc-composite-tokenizer.template:1]: Ah I see... So why just MS MARCO doc? Why not passage?

Do we want to circle back and do add composite {ms marco, dl19, dl20} x {passage, doc} in a separate PR?

Comment (theyorubayesian) [reply src/main/resources/docgen/templates/msmarco-doc-composite-tokenizer.template:1]: Okay, sure! I'll try to get it out over the weekend

Comment (theyorubayesian) [reply src/main/resources/docgen/templates/msmarco-doc-composite-tokenizer.template:1]: Do we need dl19 & dl20? I don't think we have those for `HuggingFaceTokenizerAnalyzer`

**Review by lintool — APPROVED** [review]

**Pull request merged**